store/todos.js

```diff
@@ -10,22 +10,19 @@ const getLimit = (state) => {
 
 export const state = () => ({
   list: [],
+  currentTasks: [],
   isLoading: false,
   isCreating: false,
   editableTodo: null,
   limit: pageLimit,
+  taskStates: ['all', 'complete', 'incomplete'],
+  currentTaskState: 'all',
```

Comment:
Please use `constants.js` file to host CONST variables and use them every where.
```suggestion
  import {ALL, COMPLETE, INCOMPLETE} from "@/utils/constants.js"

  taskStates: [ALL, COMPLETE, INCOMPLETE],
  currentTaskState: ALL,
```

Reply:
Nice catch. Resolved. Thanks Bhai.